src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/ReadWrite/ByteTests.cs

```diff
@@ -45,6 +45,7 @@ public void WriteByte_Pointer_Roundtrips(byte[] values)
 
         [Fact]
         [SkipOnMono("Marshal.WriteByte will not be implemented in Mono, see https://github.com/mono/mono/issues/15085.")]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/37093", TestPlatforms.Android)]
```

Comment:
Does `SkipOnMono` just apply to 'legacy' mono?

Reply:
Also, since most of these will not be implemented on Mono, I don't think an `ActiveIssue` needs to be here.  It should be truly "Skip on every Mono"

Reply:
I don't even know why it ran... since SkipOnMono should skip for all mono flavors. We should definitely look into that.

Reply:
@steveisok I think maybe the `-notrait Category=failing` is not being propagated... since we're using the HelixSdk to create our workitems we need to make sure the right `-notrait` args are propagated... I don't know if for example this run contained `-notrait Category=outerloop` which means if that is not present the outerloop tests will run and of course failures will be huge.

I don't know how will this be handled by the Helix SDK... but we usually pass those traits via ./RunTests.sh

Reply:
We do pass the correct traits but for SkipInMono it uses this reflection to check whether it's running on mono: https://github.com/dotnet/arcade/blob/e10c7cb2c0e4d9ea04705431878d16fa4268a827/src/Microsoft.DotNet.XUnitExtensions/src/DiscovererHelpers.cs#L17

It's possible that this was stripped out by the linker? We should add it to the linker xml to preserve it.

Reply:
I just tried running this test suite locally and the test wasn't executed. @mdh1418 can you please double check?

Reply:
> We do pass the correct traits but for SkipInMono it uses this reflection to check whether it's running on mono:

Btw in helix the test execution is different because the way the work items are created... I don’t think we’re passing traits there and this info was gathered from a helix run.